engines/web/app/assets/stylesheets/web/pages/cookies_policy_modal.css.scss

```diff
@@ -4,6 +4,10 @@
   background: $cookies-policy-modal-background-color;
   border-bottom-color: $cookies-policy-modal-border-bottom-color;
 
+  @media only screen and (max-width: 640px) {
+    padding: 15px 15px 0 15px;
```

Comment:
I've seen those 15px many times already. I'd be great if we could abstract it somehow. What about $gap, `$gap-sm` or something alike? we need to watch after consistency and the only way to enforce it is by sticking to a style-guide sort of vars.

Reply:
1rem is 16px right? Maybe we could use that in place if 15px here?

Reply:
assuming the font-size we set to the HTML tag is 16px, yes. And yes, that sounds more consistent.

Reply:
:+1: changed to rem